### ISSUE TITLE
Flow GoToFlow causes infinite loop on first flow

### ISSUE THREAD

**Issue by andrew-kennedy** [HaveAGitGat/Tdarr#887]
I'm having the same problem as others have mentioned before, I duplicated several flows and modified each duplicate slightly, and used go to flow between each to be able to set up a pipeline of said flows. However, whenever go to flow executes it only re-executes the first flow. All of the IDs of the various flows are correct on paper, and yet only the first flow executes. I've attached 3 of the flows in order, with their IDs so you can see the relationships:

[Remove data change container flow.json](https://github.com/HaveAGitGat/Tdarr/files/12902718/Remove.data.change.container.flow.json) with ID `ePvz2xdK6`
[Basic AV1 Flow.json](https://github.com/HaveAGitGat/Tdarr/files/12902719/Basic.AV1.Flow.json) with ID `MoqsBfAPJ`
[Lover Quality AV1 Flow.json](https://github.com/HaveAGitGat/Tdarr/files/12902720/Lover.Quality.AV1.Flow.json) with ID `rGR_J6Gld`.

Basically, from what I can tell, literally no matter what you do you will end up in an endless loop of the same original flow, even though the logs when processing a file say "go to flow" is being executed to a different flow. The subsequent steps will always be the steps from the first flow.

I'm using the latest stable build of google chrome, on Mac OS 13.5, and am running tdarr on a fedora 38 server, within a podman container using podman 4.5.

**Comment by HaveAGitGat** [comment]
Will handle this here ty:
https://github.com/HaveAGitGat/Tdarr/issues/877
